src/schemas/schemas.js

```diff
@@ -24,17 +28,73 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             "template": "../src/html/firstDiscovery.html",
             "messagePrefix": "../src/messages/",
             "message": "%prefix/firstDiscovery.json",
+            "audio": {
+                "type": "gpii.firstDiscovery.audio",
+                "panel": {
+                    "type": "gpii.firstDiscovery.panel.audio",
+                    "container": ".gpiic-fd-prefsEditor-panel-audio",
+                    "template": "%prefix/audio.html",
+                    "message": "%prefix/audio.json"
+                }
+            },
             "textSize": {
-                "type": "fluid.prefs.textSize",
-                "enactor": {
-                    "type": "fluid.prefs.enactor.textSize"
-                },
+                "type": "gpii.firstDiscovery.textSize",
                 "panel": {
-                    "type": "fluid.prefs.panel.textSize",
-                    "container": ".gpiic-size",
+                    "type": "gpii.firstDiscovery.panel.textSize",
+                    "container": ".gpiic-fd-prefsEditor-panel-size",
                     "template": "%prefix/textSize.html",
                     "message": "%prefix/textSize.json"
                 }
+            },
+            "contrast": {
+                "type": "gpii.firstDiscovery.contrast",
+                "panel": {
+                    "type": "gpii.firstDiscovery.panel.contrast",
+                    "container": ".gpiic-fd-prefsEditor-panel-contrast",
+                    "template": "%prefix/contrast.html",
+                    "message": "%prefix/contrast.json"
+                }
+            }
+        }
+    });
+
+    /*******************************************************************************
+     * Primary schema grades
+     *******************************************************************************/
+
+    fluid.defaults("gpii.firstDiscovery.schemas.textSize", {
+        gradeNames: ["autoInit", "fluid.prefs.schemas"],
+        schema: {
+            "gpii.firstDiscovery.textSize": {
+                "type": "number",
+                "default": 1,
+                "minimum": 1,
+                "maximum": 2,
+                "divisibleBy": 0.1
+            }
+        }
+    });
+
+    fluid.defaults("gpii.firstDiscovery.schemas.audio", {
+        gradeNames: ["autoInit", "fluid.prefs.schemas"],
+        schema: {
+            "gpii.firstDiscovery.audio": {
+                "type": "number",
+                "default": 1,
+                "minimum": 1,
+                "maximum": 2,
+                "divisibleBy": 0.1
+            }
+        }
+    });
+
+    fluid.defaults("fluid.prefs.schemas.contrast", {
+        gradeNames: ["autoInit", "fluid.prefs.schemas"],
```

Comment:
I've switched the text size and contrast to use the default primary schemas from infusion. The only schema we need to define here is for fluid.prefs.speak, which will be removed in the FLOE-254 branch with the update on the new infusion lib that has tts merged in.